carmajava/launch/params/HostVehicleParams.yaml

```diff
@@ -1,19 +1,23 @@
 # HostVehicleParams.yaml
 # Defines the ros parameters which define the host vehicle's characteristics
 
+# TODO it doesn't make sense for this file to contain vehicle dimensions
+# A vehicle with different dimensions would have its own urdf and therefore its own docker configuration. 
+# The only parameters in this file that would truly change between vehicles of the same configuration is ID and color. 
+
```

Comment:
I don't have a specific plan in mind, but I think we need to rethink our approach to vehicle configuration where a single file is left on the vehicle pc. Since every car now needs it's own calibration files for camera/lidar, I think we aught to make our configurations specific to each individual vehicle and develop a plan for how to manage them. I don't want to risk our calibration files getting deleted by accident. 
If we come up with an alternate approach HostVehicleParams purpose could likely change.

Reply:
I just created CAR-813 to address this need.